spec/factories/user.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :User do
+    name {Faker::name}
+    email {"VUHAISON4@GMAIL.COM"}
```

Comment:
fake toàn bộ đi em, đừng dùng dữ liệu cứng

Reply:
vâng ạ